pkg/api/kptfile/v1alpha2/validation.go

```diff
@@ -183,6 +183,20 @@ func IsKRM(n *yaml.RNode) error {
 	return nil
 }
 
+//nolint
+func AreKRM(nodes []*yaml.RNode) error {
+	for i := range nodes {
+		if err := IsKRM(nodes[i]); err != nil {
+			nodes[i].PipeE(yaml.ClearAnnotation("config.kubernetes.io/index"))
+			nodes[i].PipeE(yaml.ClearAnnotation("config.kubernetes.io/path"))
+			yaml.ClearEmptyAnnotations(nodes[i])
+			str := nodes[i].MustString()
+			return fmt.Errorf("%s:\n...\n%s...", err.Error(), str)
```

Comment:
Instead of printing the entire resource like this (which is not truncated and could be large), provide the path to the file.

Reply:
done